runtime/include/arch/aarch64/context.h

```diff
@@ -4,7 +4,6 @@
 #include <assert.h>
 
 #include "arch/common.h"
-#include "current_sandbox.h"
 
 #define ARCH_SIG_JMP_OFF 0x100 /* Based on code generated! */
```

Comment:
Can you delete this? This seems to be a zombie value.

Reply:
Sorry, do you mean `#define ARCH_SIG_JMP_OFF 0x100` is the zombie code?

Reply:
I believe so! I don't think I saw this used anywhere, but I might be wrong.